test/behaviour/graql/GraqlSteps.java

```diff
@@ -91,6 +89,13 @@ public void transaction_is_initialised() {
         assertTrue(tx.isOpen());
     }
 
+    @Given("transaction is closed and opened without commit")
+    public void transaction_is_reopened_without_commit() {
```

Comment:
We should probably align the step syntax with the method name at all times, right? Here "reopen" feels better.